src/sidebar.js

```diff
@@ -119,7 +119,7 @@ module.exports = {
           type: 'category',
           label: 'Customizing Ory',
           items: [
-            'guides/custom-domains',
+            'guides/custom-domains-and-cors',
```

Comment:
You will also need to add a rewrite if you change the URL :) An alternative would be to create another document for CORS purely.